apps/examples/testcase/le_tc/kernel/tc_sched.c

```diff
@@ -61,6 +62,38 @@ static void sched_foreach_callback(struct tcb_s *tcb, void *arg)
 }
 
 /**
+* @fn                   :function_task_setcanceltype
+* @description          :Function for tc_sched_task_setcanceltype
+* @return               :int
+*/
+#ifdef CONFIG_CANCELLATION_POINTS
+static int function_task_setcanceltype(int argc, char *argv[])
+{
+	int ret_chk;
+	int type;
+	int oldtype;
+	struct tcb_s *stcb = this_task();
+
+	type = TASK_CANCEL_INVALID;
```

Comment:
Could you leave comments at start of block to seperate negative and positive tc?